content/hardware/02.hero/boards/uno-rev3/tutorials/PWM-using-MATLAB-Tutorial/PWM-using-MATLAB-Tutorial.md

```diff
@@ -0,0 +1,148 @@
+
+![Programming successful](assets/SetupSuccess1080.PNG)
+
+
+**6.** The UNO R3 is now configured to interact with MATLAB®. Now to create the object in MATLAB®, we run the command `a=arduino()`. The properties of the object, including the COM port, are displayed as shown in the example below.
+
+![Arduino properties](assets/arduinoproperties1080.png)
+
+**7.** Enter the command `writeDigitalPin(a,'D13',1);`. This command is similar to the `digitalWrite(D13, HIGH)` by the Arduino programming language. Which means that digital pin 13 (D13) is connected to the built-in LED, `a` is the Arduino object we created, and 1 represents a HIGH or ON state. Try turning the LED ON and OFF several times by changing 1 to 0 and vice-versa.
+
+
+***Unlike in the Arduino IDE, here you cannot address the built-in LED using the `LED_BUILTIN` function.***
+
+
+**8.** Now, you can use this command to continuously blink an LED as part of a `while` loop. Try entering the script below.
+
+```
+while (1)
+      writeDigitalPin(a, 'D13', 0);
+      pause(0.5);
+      writeDigitalPin(a, 'D13', 1);
+      pause(0.5);
+end
+```
+
+**9.** See what happens when you change the pause duration. Try replacing the while with an `if` statement to make it blink a certain number of times.
+
+## Circuit
+
+On one breadboard there is a white jumper wire sending 5 volts to the potentiometer. The orange cable is connected to GND(Ground) and the gray wire is our input to A0. A0 will read the variable voltage coming in from pin13 after being adjusted through the potentiometer.
+On the right breadboard we have a simple LED circuit comprised of a red output wire from pin11, a resistor and the LED finally connected to GND via the green wire.
+
+![Full circuit of the setup.](assets/circuit.png)
+
+***Make sure to connect the longer leg of the LED to the resistor and the shorter to GND.***
+
+## Programming the Board
+
+**1.** Set your output to 1 in the previously mentioned way using `writeDigitalPin(a, 'D13', 1);`. As mentioned earlier, 1 means ON or HIGH and sends a 5V current through pin13. Check to see if A0 is actually reading an adjustable voltage using `readVoltage(a, 'A0');` in a loop and then use `display(Volts)` to print the value of `Volts`.
+
+```
+a = arduino();
+writeDigitalPin(a, 'D13', 1)
+while(1)
+  Volts = readVoltage(a, 'A0');
+  display(Volts)
+  pause(0.5)
+end
+```
+
+The amount assigned to `Volts` should be changing as you turn the potentiometer.
+
+![Voltage display](assets/Volts.PNG)
+
+**2.** Set the value of `Volts` as the voltage to be sent through pin 11 using the command: `writePWMVoltage()`. This command is different from `writeDigitalPin()` as its voltage is adjustable between 1 and 5 volts. Whereas the latter is either ON(1), being **5V** or OFF(0), **0V**.
```

Comment:
```suggestion
**2.** Set the value of `Volts` as the voltage to be set on digital pin 11 **D11** using the command: `writePWMVoltage()`. This command is adjustable between 0 and 5 volts. It will convert the value to a PWM Signal on the digital pin.
```